### ISSUE TITLE
Issue loading csv files in examples -> working directory not same as file directory for VS Code on Windows

### ISSUE THREAD

**Issue by jfrederik-nrel** [NREL/floris#873]
When I'm running the examples using VS Code through Anaconda Navigator on my Windows computer, it automatically makes `floris` (i.e., the base folder) the running directory, instead of the folder in which the example is located (i.e., `floris/examples`). As a result, when the examples (e.g., `003_wind_data_objects.py` or `006_get_farm_aep`) try to load the `inputs/wind_rose.csv` file, it can't be found because it's looking in a non-existing directory. 

This issue can easily be solved by the user adding `examples` to the path when loading the wind rose, but we probably want users on Windows to also be able to run the examples without having to change anything in the code. It also seems that someone at some point was aware of this issue, as I have noticed that `FlorisModel` does load the yaml files correctly due to this little try-except added to `floris_model.py`: 

![image](https://github.com/microsoft/vscode-python/assets/120053750/2575a8c1-a59a-403d-96db-08a0f6eb492b)

Perhaps we'll want to add a similar code snippet to other functions that load data from files, such as `WindRose.read_csv_long()` used to load `wind_rose.csv` in some of the examples.

I'm experiencing this issue on the v4 branch, but it seems to be a general problem not related to a specific version of FLORIS.

**Comment by paulf81** [comment]
This is a good catch @jfrederik-nrel , maybe the easiest solution is then to add such code to the load_csv_long function so we don't clutter the example scripts themselves?

**Comment by jfrederik-nrel** [comment]
Yes that was my thinking too. I'm just not sure how straightforward that would be and whether there are other functions that have the same issue but are not used in the example.

**Comment by misi9170** [comment]
HI @jfrederik-nrel , I presume this is still an issue? Sorry that it has taken a while to get to this. It would be nice to implement a more stable solution for windows users.

If so, we can possibly implement a fix in #1163

**Comment by misi9170** [comment]
@jfrederik-nrel pointed out that this was indeed fixed in #874, closing
